x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -783,7 +783,7 @@ FROM sample_data
 median_duration:double | client_ip:ip
 ;
 
-fieldEscaping
+fieldEscaping#[skip:-8.12.99, reason:Fixed bug in 8.13 of removing the leading/trailing backquotes of an identifier]
 FROM sample_data
 | stats count(`event_duration`) |  keep `count(``event_duration``)`
```

Comment:
@abdonpijpelink This PR introduces a (subtle) breaking change due to a previous bug:
- the leading/trailing backquotes from an identifier are removed which means that if folks used them (rare case) they would have to update their query.
```
stats count(`event_duration`) produces the alias count(``event_duration`) - same text
```
Since 8.13 (this PR) keep needs to use the same rules for quoting as the rest of commands meaning that field names with special characters need to be quoted and the backquote itself be escaped through repetition.
So 
```
count(`event_duration`)  becomes `count(``event_duration``)`
```
However that is not the case in 8.12 - the grammar rules are slightly different so the text is interpreted verbatim meaning the escaping of backticks is taken as is hence why the field is not found:
```
 Unknown column [count(``event_duration``)], did you mean [count(`event_duration`)]?
```

We need to put this information somewhere (not sure where) and in a smaller format.